assets/js/modal.js

```diff
@@ -5,6 +5,7 @@ const MODAL_SELECTOR = '#modal'
 const MODAL_CLOSE_BUTTON_SELECTOR = '#modal .modal-close'
 const MODAL_TITLE_SELECTOR = '#modal .modal-title'
 const MODAL_BODY_SELECTOR = '#modal .modal-body'
+let prevFocus = null
```

Comment:
Just a convention, but it's neat to group the mutable state in a single object, as in:

https://github.com/elixir-lang/ex_doc/blob/f5c0304f81c914c89ac6920c0c0d4110bbc6b5db/assets/js/sidebar/sidebar-drawer.js#L25-L32